common/src/main/java/org/dash/wallet/common/ui/enter_amount/EnterAmountViewModel.kt

```diff
@@ -41,7 +38,7 @@ class EnterAmountViewModel @Inject constructor(
 ) : ViewModel() {
     private val _selectedCurrencyCode = MutableStateFlow(configuration.exchangeCurrencyCode)
     var selectedCurrencyCode: String
-        get() = _selectedCurrencyCode.value
+        get() = _selectedCurrencyCode.value!!
```

Comment:
I have a concern here about the exchange rate being null, which has been the case for Krip Phones with locale of Venezuela or perhaps when the currency of the device was set to VEF.  Our app uses VES instead.   

3 months ago or so, we made some changes to prevent crashes from from null currency codes and VEF.  Perhaps there is not an issue with `!!`.  The solutions that I made previously were not robust.  I think we can have QA test for this.

Reply:
The problem is that `configuration.exchangeCurrencyCode` can return null, but it wasn't transparent for Kotlin code. So I have added a `@NotNull` annotation which then has propagated through here. But I have assumed that by this time we have a code already.

Depending on the location, I guess we can either provide a default value, or wait for changes in `exchangeCurrencyCode` the way I did it in the MainViewModel, and only act after some value is received.

Does it make sense to provide a default value here (I assume it's USD) if `configuration.exchangeCurrencyCode` is null?

Reply:
it may make sense, though it would be better that the null was handled earlier inside Configuration.